src/pandablocks/hdf.py

```diff
@@ -201,7 +198,11 @@ def mean_callable(data):
                 return (data[column_name] * field.scale / gate_duration) + field.offset
 
             return mean_callable
-        elif raw and (field.scale != 1 or field.offset != 0):
+        elif (
+            raw
+            and not field.is_pcap_bits_or_samples
+            and (field.scale != 1 or field.offset != 0)
+        ):
```

Comment:
Could: I'd prefer if the function was named after what we wanted from it, rather than what it is, and we always scaled even if it was 1
```suggestion
        elif raw and field.has_scale_and_offset:
```